.circleci/config/base.yml

```diff
@@ -682,7 +655,7 @@ step-persist-data-for-tests: &step-persist-data-for-tests
       - src/out/Default/dist.zip
       - src/out/Default/mksnapshot.zip
       - src/out/Default/chromedriver.zip
-      - src/out/Default/shell_browser_ui_unittests
+      # - src/out/Default/shell_browser_ui_unittests
```

Comment:
Not sure if we want to keep this commented out (also on L1260), but could we just delete this instead of commenting it out if we don't need it?